client/src/components/feedback/index.js

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from 'react';
+import ReactStars from 'react-stars';
+import propTypes from 'prop-types';
+import './style.css';
+import Button from '../utils/Button';
+
+export default class FeedbackComponent extends Component {
+  state = {
+    rate: 2.5,
+    email: '',
+    content: '',
+    errorMessage: '',
+  };
+
+  setFeedback = event => this.setState({ content: event.target.value });
+
+  setEmail = event => this.setState({ email: event.target.value });
+
+  newRate = newR => this.setState({ rate: newR });
+
+  sendFeedback = () => {
+    const { history, orderId = 1 } = this.props;
+    const { email, content: feedback } = this.state;
+    const data = {
+      orderId,
+      email,
+      feedback,
+    };
+    fetch('/api/v1/post-feedback', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify(data),
+    })
+      .then(res => res.json())
+      .then(status => {
+        if (status.statusCode === 201) {
+          this.setState({ errorMessage: '' });
+          history.push('/home');
+        } else {
+          this.setState({ errorMessage: status.error });
+        }
+      })
+      .catch(err => {
+        this.setState({ errorMessage: err });
+      });
+  };
+
+  render() {
+    const { history } = this.props;
+    const { rate, errorMessage } = this.state;
+    return (
+      <div className="bg">
+        <div className="card">
+          <h3 className="header-feedback"> feedback</h3>
+          <div className="order-info">
+            <div className="order-number">
+              <p> 23</p>
+              <h4>Order</h4>
+            </div>
+            <div className="table-number">
+              <p>5</p>
+              <h4>Table</h4>
+            </div>
+          </div>
+          <div>{errorMessage}</div>
+          <form>
+            <input
+              type="text"
+              onChange={this.setEmail}
```

Comment:
Use the full definition of `controlled components` with your inputs, which means your state **must** always be your only source of truth.
https://reactjs.org/docs/forms.html#controlled-components